app/app/vendor/controllers/site-ctrl.js

```diff
@@ -2,6 +2,7 @@
 
 var SiteCtrl = function ($rootScope) {
   const config = require('../../config/application.json');
+  const teste = '../../config/locale/' + config.locale + '.json';
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'teste' is defined but never used.